### ISSUE TITLE
mkfs.btrfs: command not found

### ISSUE THREAD

**Issue by brianmcgee** [nix-community/disko#86]
I'm using this config:

```nix
{ disks ? [ "/dev/nvme0n1" ], ... }: {
  disk = {
    vdb = {
      type = "disk";
      device = builtins.elemAt disks 0;
      content = {
        type = "table";
        format = "gpt";
        partitions = [
          {
            type = "partition";
            name = "ESP";
            start = "1MiB";
            end = "512MiB";
            bootable = true;
            content = {
              type = "filesystem";
              format = "vfat";
              mountpoint = "/boot";
              options = [
                "defaults"
              ];
            };
          }
          {
            type = "partition";
            name = "luks";
            start = "512MiB";
            end = "100%";
            content = {
              type = "luks";
              name = "crypted";
              keyFile = "/tmp/secret.key";
              content = {
                type = "btrfs";
                mountpoint = "/";
                subvolumes = [
                  "/home"
                ];
              };
            };
          }
        ];
      };
    };
  };
}
```

And running it as follows:

```shell
[root@nixos:~]# nix run github:nix-community/disko --no-write-lock-file -- -m create ./luks-btrfs.nix 
warning: not writing modified lock file of flake 'github:nix-community/disko':
• Added input 'nixpkgs':
    'path:/nix/store/dk1bqlz87dzsd8g4fq92pscgd60ji2vr-source?lastModified=0&narHash=sha256-n5UBO6XBV4h3TB7FYu2yAuNQMEYOrQyKeODUwKe06ow=' (1970-01-01)
this derivation will be built:
  /nix/store/32al7lhqnjby2cjymdlr6l1vr9wy3yir-disko-create.drv
building '/nix/store/32al7lhqnjby2cjymdlr6l1vr9wy3yir-disko-create.drv'...
+ parted -s /dev/nvme0n1 -- mklabel gpt
+ parted -s /dev/nvme0n1 -- mkpart ESP 1MiB 512MiB
+ udevadm trigger --subsystem-match=block
+ udevadm settle
+ parted -s /dev/nvme0n1 -- set 1 boot on
+ udevadm trigger --subsystem-match=block
+ udevadm settle
+ mkfs.vfat /dev/nvme0n1p1
mkfs.fat 4.2 (2021-01-31)
+ parted -s /dev/nvme0n1 -- mkpart luks 512MiB 100%
+ udevadm trigger --subsystem-match=block
+ udevadm settle
+ udevadm trigger --subsystem-match=block
+ udevadm settle
+ cryptsetup -q luksFormat /dev/nvme0n1p2 /tmp/secret.key
+ cryptsetup luksOpen /dev/nvme0n1p2 crypted --key-file /tmp/secret.key
+ mkfs.btrfs /dev/mapper/crypted
/nix/store/ji7jbvlz22yycysdg0i75a6xan4l6hlg-disko-create: line 30: mkfs.btrfs: command not found
```

Any ideas why the script seems to think `mkfs.btrfs` isn't available?

**Comment by lilyinstarlight** [comment]
I had the same issue. This should be fixed by #87 (or #82 as well)

**Comment by Mic92** [comment]
should be fixed now.
